views/artist-search.hbs

```diff
@@ -0,0 +1,11 @@
+{{#each allResults}}
+
+  <div>
+    <p>{{this.name}}</p>
+
+    <img src="{{this.images.0.url}}" alt="artist-img" />
+
+    <a href="/albums"><button>View Albums!</button></a>
```

Comment:
Here, to navigate to the specific album, you must pass its id. 
`<a href="/albums/{{this.id}}"><button>View Albums!</button></a>`